module_utils/kafka_lib_user.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+import collections
+import traceback
+
+from kafka.errors import KafkaError
+
+from ansible.module_utils import kafka_scram
+from ansible.module_utils.pycompat24 import get_exception
+from ansible.module_utils.kafka_lib_commons import (
+    get_manager_from_params,
+    maybe_clean_kafka_ssl_files,
+    maybe_clean_zk_ssl_files
+)
+
+
+module_user_spec_commons = dict(
+    name=dict(
+        type='str',
+        required=True
+    ),
+    password=dict(
+        type='str',
+        required=False,
+        no_log=True
+    ),
+    mechanism=dict(
+        choices=['SCRAM-SHA-256', 'SCRAM-SHA-512'],
+        default='SCRAM-SHA-512'
+    ),
+    iterations=dict(
+        type='int',
+        required=False
+    ),
+    state=dict(
+        choices=['present', 'absent', 'update'],
```

Comment:
If i understand 'update' force user to be updated. Might prefer 'updated' to have a more ansible way.